tests/test_ci/TestMarkdown.py

```diff
@@ -0,0 +1,128 @@
+from tests.base import BaseTestCase
+
+from markdown2 import markdown
+
+
+class TestMarkdown(BaseTestCase):
+
+    def test_heading(self):
```

Comment:
I'd personally split it up into two separate tests. If one fails you see immediately which one :)

Reply:
I don't think so; there are two tests just to ensure consistency of the heading tag. Otherwise, we should test till h6, but just to ensure consistency we are going for two. Hence I think it will only make it redundant to write it twice.

Reply:
Also makes sense :)